PhysicsTools/Heppy/python/physicsobjects/Lepton.py

```diff
@@ -44,6 +44,23 @@ def relIso(self,dBetaFactor=0, allCharged=0):
         rel = self.absIso(dBetaFactor, allCharged)/self.pt()
         return rel
 
+    def absIsoR(self, R=0.3, dBetaFactor=0, allCharged=False):
+        if dBetaFactor>0 and self.puChargedHadronIsoR(R)<0:
+            raise ValueError('If you want to use dbeta corrections, you must make sure that the pu charged hadron iso is available. This should never happen') 
+        neutralIso = self.neutralHadronIsoR(R) + self.photonIsoR(R)
+        #Recover FSR
+        if hasattr(self, 'fsrPhotons'):
+            for gamma in self.fsrPhotons:
+                neutralIso = neutralIso - gamma.pt()
```

Comment:
Yes, I'd be in favour of dropping the old absIso and just have your new absIsoR.
Nobody is using the old absIso anywhere, at least from what github search reveals.